samples/v1.2/Scenarios/FlightDetails.json

```diff
@@ -160,7 +160,7 @@
 											"items": [
 												{
 													"type": "TextBlock",
-													"text": "November 12, 2017",
+													"text": "Sat, Mar 4, 2017",
```

Comment:
Shouldn't we have to update this textblocks to use {Date} formatting in the string? Seems like we're missing an opportunity here

Reply:
@almedina-ms with @RebeccaAnne out can you pick up where she left off here to shepherd this change in? Thanks!

Reply:
I think we should probably fix the date stuff later rather than hold up the release. I'll file a new issue.

Reply:
filed #4709